docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java

```diff
@@ -66,8 +66,8 @@ public DocsClientYamlTestSuiteIT(@Name("yaml") ClientYamlTestCandidate testCandi
         entries.addAll(ExecutableSection.DEFAULT_EXECUTABLE_CONTEXTS);
         entries.add(new NamedXContentRegistry.Entry(ExecutableSection.class,
                 new ParseField("compare_analyzers"), CompareAnalyzers::parse));
-        NamedXContentRegistry executeableSectionRegistry = new NamedXContentRegistry(entries);
-        return ESClientYamlSuiteTestCase.createParameters(executeableSectionRegistry);
+        NamedXContentRegistry executableSectionRegistry = new NamedXContentRegistry(entries);
```

Comment:
This is now the only file that's a `.java` -- it's a bit odd that it lives in `docs/`

Reply:
Got it, I think it's there because it's the test class that tests the docs snippets. I think it makes sense for it to be there. This way when running ./gradlew :docs:check that one gets run.